slynk/slynk.lisp

```diff
@@ -3219,7 +3241,9 @@ QUALIFIERS and SPECIALIZERS are lists of strings."
            (mapcar (lambda (specializer)
                      (if (typep specializer 'slynk-mop:eql-specializer)
                          (format nil "(eql ~A)"
-                                 (slynk-mop:eql-specializer-object specializer))
+                                 (funcall #+allegro 'mop:eql-specializer-object
```

Comment:
Is it intended that this will only work on Allegro or SBCL?  I don't know enough about the context.